sklearn/datasets/tests/test_openml.py

```diff
@@ -1459,7 +1459,7 @@ def _mock_urlopen_raise(request, *args, **kwargs):
         (False, "pandas"),
     ],
 )
-def test_fetch_openml_verify_checksum(monkeypatch, as_frame, cache, tmpdir, parser):
+def test_fetch_openml_verify_checksum(monkeypatch, as_frame, tmpdir, parser):
```

Comment:
Here, `cache` was a spurious argument that was unused throughout the file. It broke the Pyodide tests and returned an error upon collection because the interpreter assumed that this was a fixture.